developers_affiliations7.txt

```diff
@@ -15605,12 +15605,7 @@ sk-: sk-!users.noreply.github.com
 sk4zuzu: mopala!opennebula.io
 	Independent
 sk593: shruthikumar2401!gmail.com, sk593!users.noreply.github.com
-	Independent until 2020-05-01
```

Comment:
I don't recognize these companies or dates. Wondering if there was a mixup in users somewhere?

Reply:
Don't know, sorry, I'll update according to this PR.